nemo/collections/tts/models/fastpitch.py

```diff
@@ -217,6 +217,12 @@ def _setup_normalizer(self, cfg):
                 raise ImportError(
                     "`nemo_text_processing` not installed, see https://github.com/NVIDIA/NeMo-text-processing for more details"
                 )
+            # fmt: off
+            import pkg_resources
+            # safeguard to the version 0.1.8rc0 to avoid unexpected normalized outputs for Chinese. Will remove the guard once
+            # next-text-processing new release fixes the bug. Tracking bugfix in https://github.com/NVIDIA/NeMo-text-processing/issues/109.
+            pkg_resources.require("nemo_text_processing==0.1.8rc0")
```

Comment:
Does this raise an error if the version is incorrect? If so, can you print an appropriate error message for why this is pinned?

Reply:
yes, it does,
```
ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_from_pretrained_ctc_model - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_from_pretrained_rnnt_model - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_from_asr_config - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_save_restore - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_save_restore_asr - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/asr/test_hybrid_asr_tts_models.py::TestASRWithTTSModel::test_from_pretrained_ctc_model_fused_bn - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))

ERROR tests/collections/tts/test_tts_exportables.py::TestExportable::test_FastPitchModel_export_to_onnx - pkg_resources.VersionConflict: (nemo-text-processing 0.2.0rc0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('nemo_text_processing==0.1.8rc0'))
```